torch/nn/__init__.py

```diff
@@ -3,3 +3,9 @@
 from .parallel import DataParallel
 from . import init
 from . import utils
```

Comment:
These lines above (github won't let me select them) appear to also only exist to reexport things, but I'm not entirely certain as an outsider... 🙂